htdocs/core/triggers/interface_50_modTicket_TicketEmail.class.php

```diff
@@ -461,4 +470,62 @@ private function composeAndSendCustomerMessage($sendto, $base_subject, $body, $s
 			$conf->global->MAIN_MAIL_AUTOCOPY_TO = $old_MAIN_MAIL_AUTOCOPY_TO;
 		}
 	}
+
+	/**
+	 * Composes and sends a message concerning a ticket, to be sent to user assigned to the ticket
+	 *
+	 * @param string 	$sendto			Addresses to send the mail, format "first@address.net, second@address.net, " etc.
+	 * @param string 	$base_subject	email subject. Non-translated string.
+	 * @param string	$body			email body (first line). Non-translated string.
+	 * @param string 	$see_ticket		string indicating the ticket public address
+	 * @param Ticket 	$object			the ticket thet the email refers to
+	 * @param Translate $langs			the translation object
+	 * @return void
+	 */
+	private function composeAndSendAssigneeMessage($sendto, $base_subject, $body, $see_ticket, Ticket $object, Translate $langs)
+	{
+		global $conf, $user;
+
+		// Init to avoid errors
+		$filepath = array();
+		$filename = array();
+		$mimetype = array();
+
+		// Send email to assigned user
+		$subject = '['.$conf->global->MAIN_INFO_SOCIETE_NOM.'] '.$langs->transnoentities($base_subject);
```

Comment:
Old code was using
'['.$mysoc->name.']'
why changing into 
$conf->global->MAIN_INFO_SOCIETE_NOM  ?

Reply:
Because it wasn't passed as global, I just changed it.